utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/delayed/DelayedKafkaSourceFactory.scala

```diff
@@ -82,7 +83,7 @@ object DelayedKafkaSourceFactory {
   def validateTimestampField(field: String, typingResult: TypingResult)(implicit nodeId: NodeId): List[ProcessCompilationError] = {
     typingResult match {
       case TypedObjectTypingResult(fields, _, _) => fields.get(field) match {
-        case Some(fieldTypingResult) if List(Typed[java.lang.Long], Typed[Long]).contains(fieldTypingResult) => List.empty
+        case Some(fieldTypingResult) if List(Typed[java.lang.Long], Typed[Long], Typed[Int], Typed[Instant], Typed[LocalDate], Typed[ZonedDateTime]).contains(fieldTypingResult) => List.empty
```

Comment:
It would be nice to have this list somewhere near runtime (`supportedTypeToMillis`) so it will be harder to desynchronize this code